private/prepare_mesh_tetrahedral.m

```diff
@@ -86,11 +86,12 @@
 % this requires the external iso2mesh toolbox
 ft_hastoolbox('iso2mesh', 1);
 
-[node, elem, face] = vol2mesh(seg, 1:mri.dim(1), 1:mri.dim(2), 1:mri.dim(3), 2, 2, 0, 'cgalsurf');
+[node, elem, face] = vol2mesh(uint8(seg), 1:mri.dim(1), 1:mri.dim(2), 1:mri.dim(3), 1, 1, 0, 'cgalmesh');
+elem(:,1:4)        = meshreorient(node(:,1:3), elem(:,1:4));
 
 mesh = keepfields(mri, {'coordsys', 'unit'});
-mesh.pos = ft_warp_apply(mri.transform, node);
-mesh.tet = elem(:,[1 2 4 3]); % re-order elements
```

Comment:
Hmm, I hope @schoffelen carefully considered that the reordering of elements was not needed any more. By just looking at the code, it is not possible to see whether this is a proper change.

Reply:
Yes, I did. The reordering is now done in meshorient, which gets the orientation consistent. sb_test_ori now passes (which it didn't before, even with the magical [1 2 4 3] swap, because with the new mesh generation the orientation of the tetraeders was totally random (mix of left and right handed elements)). sb_test_ori kind of assumes all elements to be either left or right handed.